android-appcoins-billing/src/main/java/com/appcoins/sdk/billing/helpers/EventLogger.java

```diff
@@ -0,0 +1,105 @@
+package com.appcoins.sdk.billing.helpers;
+
+import com.appcoins.billing.sdk.BuildConfig;
+
+import org.json.JSONException;
+import org.json.JSONObject;
+
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStreamReader;
+import java.io.OutputStream;
+import java.io.OutputStreamWriter;
+import java.net.HttpURLConnection;
+import java.net.MalformedURLException;
+import java.net.ProtocolException;
+import java.net.URL;
+
+import javax.net.ssl.HttpsURLConnection;
+
+public class EventLogger {
+
+    public static String ENTITY_NAME = "SDK";
+
+    private final String BASE_URL = "https://ws75.aptoide.com/api/7/";
```

Comment:
Pass this URL to build config

Reply:
I like the way of thinking of this suggestion but I think that this var is only relevant to eventLogger